applications/StructuralMechanicsApplication/custom_elements/solid_elements/base_solid_element.cpp

```diff
@@ -747,6 +747,7 @@ void BaseSolidElement::CalculateOnIntegrationPoints(
 
     if ( rOutput.size() != number_of_integration_points )
         rOutput.resize( number_of_integration_points, false );
```

Comment:
```suggestion
        rOutput.resize( number_of_integration_points);
```
Or 0.0 or simply remove it if we do assign later

Reply:
yes, the assing not only puts values but also resizes. I remove the resize